templates/dokumenter.html

```diff
@@ -8,14 +8,20 @@
 {% block content %}
 <h1>{{ section.title }}</h1>
 
-
-{% for x in section.subsections %}
-  {% set y =  x | trim_end_matches(pat="/_index.md") %}
-  {% set z =  y | trim_start_matches(pat="dokumenter/") %}
+<!-- Trims the page link into a single word (name of document). -->
+{% for filePath in section.subsections %}
+  {% set prevFilePath = filePath | trim_end_matches(pat="/_index.md") %}  
+  {% set dokuType = prevFilePath | trim_start_matches(pat="dokumenter/") | capitalize %}
   
-  <ul>
-    <td><a href="{{y}}" target="_blank" class="footer--link">{{z}}</a></td>
+
+  {% set subsection = get_section(path=filePath) %}
+
+  <ul>  
+    <!-- <td><a href="{{filePath}}" target="_blank" class="footer--link">{{dokuType}}</a></td> -->
+    <td><a href="{{ get_url(path=subsection.path) }}">{{ subsection.title }}</a></td>
```

Comment:
```suggestion
    <li><a href="{{ get_url(path=subsection.path) }}">{{ subsection.title }}</a></li>
```

`td`-tagg er forbeholdt tabeller, se [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td).